crates/fluvio-storage/Cargo.toml

```diff
@@ -56,6 +58,7 @@ fluvio-future = { workspace = true, features = ["fixture"] }
 flv-util = { workspace = true, features = ["fixture"] }
 fluvio-socket = { workspace = true,  features = ["file"] }
 fluvio-protocol = { workspace = true,  features = ["fixture"] }
+fluvio-compression = { workspace = true, features = ["compress"] }
```

Comment:
is this only needed for iterator? if so, make it optional and add to iterators's dependency

Reply:
It is not needed for iterator, It is just used in tests. I added this in `dev-dependencies` only